.github/CODEOWNERS

```diff
@@ -250,82 +236,82 @@
-/core/templates/services/stateful/focus-manager.service*.ts @srijanreddy98
-/core/templates/services/contextual/device-info.service*.ts @srijanreddy98
-/core/templates/services/contextual/url.service*.ts @srijanreddy98
-/core/templates/services/contextual/window-dimensions.service*.ts @srijanreddy98
-/core/templates/services/contextual/window-ref.service*.ts @srijanreddy98
-/core/templates/services/internet-connectivity.service*.ts @srijanreddy98
-/core/templates/services/server-connection-backend-api.service*.ts @srijanreddy98
-/core/templates/services/number-conversion.service*.ts @srijanreddy98
+/core/templates/services/alerts.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/date-time-format.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/debouncer.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/extension-tag-assembler.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/html-escaper.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/id-generation.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/keyboard-shortcut.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/local-storage.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/nested-directives*.ts @oppia/angular-migration-reviewers
+/core/templates/services/page-title.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/prevent-page-unload-event.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/promo-bar-backend-api.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/site-analytics.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/utils.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/stateful/background-mask.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/stateful/focus-manager.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/contextual/device-info.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/contextual/url.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/contextual/window-dimensions.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/contextual/window-ref.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/internet-connectivity.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/server-connection-backend-api.service*.ts @oppia/angular-migration-reviewers
+/core/templates/services/number-conversion.service*.ts @oppia/angular-migration-reviewers
 
 
 # Global stylesheet.
-/core/templates/css/oppia.css @srijanreddy98
-/core/templates/css/oppia-material.css @srijanreddy98
+/core/templates/css/oppia.css @oppia/angular-migration-reviewers
+/core/templates/css/oppia-material.css @oppia/angular-migration-reviewers
```

Comment:
Perhaps this is better served with https://github.com/orgs/oppia/teams/frontend-infrastructure-reviewers as they will stay even after the migration here.

Reply:
This is fine with me. @ashutoshc8101 any objections (since you initially suggested assigning these to angular-migration-reviewers)?

Reply:
done